mls-rs-uniffi/src/lib.rs

```diff
@@ -17,6 +17,9 @@
 //!
 //! [UniFFI]: https://mozilla.github.io/uniffi-rs/
 
+#[cfg(test)]
```

Comment:
Ah, I see how that you use a pattern of having a `test_util` Cargo feature for this. I can update to use the same pattern here.

Reply:
I think what you did is simpler? AFAIK we only used the feature when a `test_util` module was used in different crates. E.g. we have "test_util" feature for HPKE so that different crypto provider crates using the HPKE crate can execute known-answer HPKE tests

Reply:
> I think what you did is simpler?

I guess: I had not thought about this being a matter of cross-crate dependencies. I'm happy to leave it like it is now if you think it's fine.

Reply:
Thanks!